Guide-to-migrate-e2e-tests.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Currently, Oppia E2E tests are in a hybrid state where we have both WebdriverIO and Protractor test suites. The project **Migrate away from Protractor** aims to migrate all the existing E2E tests to WebdriverIO.
+
+This guide is to help contributors write/modify E2E tests in webdriverIO during the hybrid state.
+
+## Selectors
+
+The `$` command is a short way to call the findElement command in order to fetch a single element on the page
+
+| Name                                     | Protractor                               |WebdriverIO           |
+| -----------------------------------------| -----------------------------------------|----------------------|
+| By class name                            |`element(by.css('.className'))`           |`$('.className')`     |
```

Comment:
It's unclear that `.className` should be replaced with the class name the user wants to select. Ditto throughout these tables--it's unclear what text the user is supposed to replace

Reply:
Updated.